### ISSUE TITLE
CEdit may not handle variable vent openings with only one input

### ISSUE THREAD

**Issue by rpeacoc** [firemodels/cfast#1935]
From a discussion group user:

I have encountered a potential issue with time-based vent openings.  This issue appears in both CFAST 7.7.1 and 7.6.0 (which I was running comparisons against) and occurs with a single or multiple compartment-to-compartment vents.

It seems that if only one Open/Close Criterion Table entry is entered in using the CEdit GUI, the program doesn’t factor the input into the calculation and doesn’t save the input to the .in file.
I can get the input to be reflected in the model and saved in the .in file by duplicating the desired value into the Open/Close Criterion Table (i.e. two identical rows for time and opening fraction). 
Also, if I manually open the .in file in a text editor and enter in just one table entry, the model will run correctly and save the input in the .in file. 
I could not find text in the user’s manual or technical reference to indicate duplicate entries are required.

**Comment by mcgratta** [comment]
Is this still an issue?

**Comment by rpeacoc** [comment]
Yes, but it’s an easy workaround to just have two inputs.Sent from my iPhoneOn Apr 10, 2026, at 3:38 PM, Kevin McGrattan ***@***.***> wrote:﻿mcgratta left a comment (firemodels/cfast#1935)
Is this still an issue?

**Comment by mcgratta** [comment]
Is there some text we can add to the User's Guide?

**Comment by rpeacoc** [comment]
Sure. I'll take a look. We could also add a warning or error message.

**Comment by mcgratta** [comment]
This issue will be addressed in the new GUI for CFAST.

**Comment by rmcdermo** [comment]
People... help me out here.  Where is the input file that does not work?

**Comment by mcgratta** [comment]
Let me see if I can break the new GUI. If not, I'll close the issue.

**Comment by mcgratta** [comment]
I opened the new GUI, added a single Open/Close table, and the new editor saved it properly.
